docs/openapi/components/schemas/common/DeliverySchedule.yml

```diff
@@ -0,0 +1,199 @@
+$linkedData:
+  term: DeliverySchedule
+  '@id': https://w3id.org/traceability#DeliverySchedule
+title: Delivery Schedule
+description: A set of data used to plan transportation of commodities from receipt facilities to delivery facilities
+type: object
+properties:
+  type:
+    oneOf:
+      - type: array
+        items:
+          type: string
+          enum:
+            - DeliverySchedule
+      - type: string
+        const: DeliverySchedule
+  pipeline:
```

Comment:
Should this change to the generic organization instead of being a pipeline? 
This should be as generic a Delivery schedule as possible

Reply:
updated `pipeline` to `transporter`